bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/NuvoBindingConstants.java

```diff
@@ -70,20 +71,41 @@ public class NuvoBindingConstants {
     public static final String TYPE_SOURCE_UPDATE = "source_update";
     public static final String TYPE_ZONE_UPDATE = "zone_update";
     public static final String TYPE_ZONE_BUTTON = "zone_button";
+    public static final String TYPE_ZONE_BUTTON2 = "zone_button2";
+    public static final String TYPE_ZONE_MENUREQ = "zone_menureq";
+    public static final String TYPE_MENU_ITEM_SELECTED = "top_menu_button";
     public static final String TYPE_ZONE_CONFIG = "zone_config";
+    public static final String TYPE_ALBUM_ART_REQ = "album_art_req";
+    public static final String TYPE_ALBUM_ART_FRAG_REQ = "album_art_frag_req";
+    public static final String TYPE_FAVORITE_REQ = "favorite_req";
 
     // misc
     public static final String ON = "ON";
     public static final String OFF = "OFF";
+    public static final String TWO = "2";
     public static final String ONE = "1";
     public static final String ZERO = "0";
     public static final String BLANK = "";
+    public static final String SPACE = " ";
+    public static final String COMMA = ",";
     public static final String DISPLINE = "DISPLINE";
     public static final String DISPINFO = "DISPINFO,"; // yes comma here
+    public static final String DISPINFOTWO = "DISPINFOTWO";
     public static final String NAME_QUOTE = "NAME\"";
     public static final String MUTE = "MUTE";
     public static final String VOL = "VOL";
+    public static final String OX = "0x";
```

Comment:
Can this be changed to a description of what it does?  Is a delimiter, end of line, etc?

Reply:
Changed to be more descriptive.